_posts/en/2018-12-28-optech-review-2018-plan-2019.md

```diff
@@ -0,0 +1,79 @@
+---
+title: 'Optech review of 2018 and plans for 2019'
+permalink: /en/optech-review-2018-plan-2019
+name: 2018-12-28-optech-review-2018-plan-2019
+type: posts
+layout: post
+lang: en
+version: 1
+---
+This month, Steve, Mike, James and I met to review the progress that Optech made
```

Comment:
There's no authorship to this post, so "I" is ambiguous.  I suggest using your name and combining this paragraph with the following paragraph as that introduces who y'all are.

Reply:
Just changed this to 'the optech team', with a link to the about page of our website.